drivers/flash/flash_shell.c

```diff
@@ -255,6 +255,158 @@ static int cmd_test(const struct shell *shell, size_t argc, char *argv[])
 	return result;
 }
 
+static int set_bypass(const struct shell *shell, shell_bypass_cb_t bypass)
```

Comment:
This will fail CI, you need to change it to:
`static int set_bypass(const struct shell *sh, shell_bypass_cb_t bypass)`